### PR TITLE
feat: clubs

### PR DESCRIPTION
**Pull request by KambojRajan**

### CODE REVIEW

**Review by geekysilento — CHANGES_REQUESTED** [review]
image
- The image bleeds out of its div
- The Heading of the club page is h1, so are all the subheadings, please make subheadings smaller.

**Review by GetPsyched — CHANGES_REQUESTED** [review]
is this still a work in progress? I only see About and RSVP heading. The design has a lot more stuff.

Also what is RSVP here?

**Comment by KambojRajan** [comment]
We have a design for this? Additionally, there is little to no data available for many of the clubs(in the current website). For clubs that have complete data, the components will display properly. for me this is not wip. The RSVP section will include the names and contact information of either the students in the club or the secretary if you want to name it some other way I will do that.

**Comment by GetPsyched** [comment]
The design exists, it's not in the Dev Handoff page, but in the one where everything was, see the left sidebar.

For the data, just fetch from DB and then use `??` to fallback on dummy data that you can put. This way the type of the dict remains exactly what the db is providing us with. @geekysilento can give you an example if needed.

File: drizzle.config.ts [hunk 1]
```diff
@@ -6,8 +6,10 @@ export default defineConfig({
   schema: './server/db/schema/**/*.schema.ts',
   dialect: 'postgresql',
   dbCredentials: {
-    url: env.POSTGRES_URL,
+    url: "postgresql://$default:1tpVmZYrd0FG@ep-soft-limit-a1lqwhsh-pooler.ap-southeast-1.aws.neon.tech:5432/verceldb?sslmode=require",
```
Comment (heydoyouknowme0) [inline drizzle.config.ts:9]: why?

Comment (heydoyouknowme0) [reply drizzle.config.ts:9]: @KambojRajan

**Pull request closed without merging**